src/utils.py

```diff
@@ -0,0 +1,65 @@
+import mysql.connector
+
+
+def dbconnect(
+    hostname: str,
+    username: str,
+    dbname:str):
+    """
+    Method to connect database and python Script
+    
+    Parameters
+    ----------
+    hostname : (str) host name of mysql-server 
+    username : (str) username of mysql-server
+    dbname : (str) name of database to be accessed 
+    
+    Returns
+        Returns Tuple(con,cr) connection and cursor
+    """
+    
+    con = mysql.connector.connect(host=hostname,
+                                  user=username,
+                                  password = 'password',
+                                  database = dbname)
```

Comment:
use `dotenv` library to store credentials.

```
from dotenv import load_dotenv
load_dotenv(".env")
```

then replace `password = os.getenv("PASSWORD")`

in `.env` file, 
`export PASSWORD=your_password`